<ui version="4.0" >
 <class>messagingSettingsClass</class>
 <widget class="QWidget" name="messagingSettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>messagingSettings</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="tabBox" >
        <property name="text" >
         <string>Tabbed messaging mode</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="chatNamesBox" >
        <property name="text" >
         <string>Show names</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="timeLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Timestamp:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="timeComboBox" >
          <property name="currentIndex" >
           <number>1</number>
          </property>
          <item>
           <property name="text" >
            <string>( hour:min:sec day/month/year )</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>( hour:min:sec )</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>( hour:min, full date )</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="onEnterBox" >
        <property name="text" >
         <string>Send message on enter</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QCheckBox" name="closeTabBox" >
        <property name="text" >
         <string>Close tab/message window after sending a message</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QCheckBox" name="eventBox" >
        <property name="text" >
         <string>Don't show events if message window is open</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QCheckBox" name="openNewBox" >
        <property name="text" >
         <string>Open message window/tab if received message</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="codepageLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Codepage:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="codepageBox" >
          <property name="currentIndex" >
           <number>47</number>
          </property>
          <item>
           <property name="text" >
            <string>Apple Roman</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Big5</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Big5-HKSCS</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>EUC-JP</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>EUC-KR</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>GB18030-0</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>IBM 850</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>IBM 866</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>IBM 874</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 2022-JP</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-1</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-2</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-3</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-4</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-5</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-6</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-7</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-8</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-9</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-10</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-13</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-14</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-15</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ISO 8859-16</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Iscii-Bng</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Iscii-Dev</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Iscii-Gjr</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Iscii-Knd</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Iscii-Mlm</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Iscii-Ori</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Iscii-Pnj</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Iscii-Tlg</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Iscii-Tml</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>JIS X 0201</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>JIS X 0208</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>KOI8-R</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>KOI8-U</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>MuleLao-1</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ROMAN8</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Shift-JIS</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>TIS-620</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>TSCII</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>UTF-8</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>UTF-16</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>UTF-16BE</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>UTF-16LE</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Windows-1250</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Windows-1251</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Windows-1252</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Windows-1253</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Windows-1254</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Windows-1255</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Windows-1256</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Windows-1257</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Windows-1258</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>WINSAMI2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0" >
       <widget class="QCheckBox" name="typingBox" >
        <property name="text" >
         <string>Send typing notifications</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" >
       <widget class="QCheckBox" name="lightChatBox" >
        <property name="text" >
         <string>Chat view light version</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" >
       <widget class="QCheckBox" name="hideMessageBox" >
        <property name="text" >
         <string>Don't show messages in tray events</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
